modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -219,12 +219,18 @@ definition {
 	}
 
 	@description = "Verify that the user can add Nodes into diagram by drag&drop: Task"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAddTaskNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanAddTaskNode";
+
+		Workflow.openWorkflowListView();
 
-		// TODO CanAddTaskNode pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Task",
+			nodePosition = "-20,-40",
+			nodeType = "task");
 	}
```

Comment:
It would be interesting to add a `viewPanelNodePresent` to check the node in the diagram.